docs/embedded/concepts/content-experiences/search-content.md

```diff
@@ -10,11 +10,12 @@ ms.localizationpriority: high
 Use the Microsoft Search API in Microsoft Graph to search SharePoint Embedded containers and content. The Search API lets you scope the container type and file type for your queries by specifying the corresponding parameter in the request body. This article describes some examples.
 
 > [!NOTE]
-> Search API only supports Delegated permissions.
+> 1. Search API only supports Delegated permissions.
```

Comment:
Markdown numbered bullets should always use 1.... don't use explicit numbers. Let the rendering engine do the numbering for you.

Bullets should be separated by a newline... in other words, there should be a blank line between lines 35 & 36